pysat/_orbits.py

```diff
@@ -344,6 +344,8 @@ def _polarBreaks(self):
             # print 'Time Gap'
 
         # create orbitbreak index, ensure first element is always 0
+        if len(ind) == 0:
+            ind = np.array([0])
```

Comment:
Can you add a unit test for the 0 length case?

Reply:
Lines covered in latest build.  Thanks!

https://coveralls.io/builds/24314833/source?filename=pysat%2F_orbits.py